lib/src/progress_tracker_widget.dart

```diff
@@ -25,6 +25,8 @@ class ProgressTracker extends StatelessWidget {
   /// Defaults to [Colors.grey] if not specified.
   final Color? inActiveColor;
 
+  final double height = 100;
```

Comment:
Patch for text with 2 lines.
This should be fixed without setting constheight to the progress tracker.